data/ext/pending/issue-2636.ttl

```diff
@@ -0,0 +1,12 @@
+@prefix : <http://schema.org/> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+
+:eduQuestionType a rdf:Property ;
+    rdfs:label "eduQuestionType" ;
+    :category "issue-2636" ;
+    :domainIncludes :Question ;
+    :isPartOf <http://pending.schema.org> ;
+    :rangeIncludes :Text ;
+    :source <https://github.com/schemaorg/schemaorg/issues/2636> ;
+    rdfs:comment "For questions that are part of learning resources (e.g. Quiz), eduQuestionType indicates the format of question being given. Example: "Multiple choice", "Open ended", "Flashcard"." .
```

Comment:
The Travis-CI checks are failing on this as it is not well-formed Turtle: you're using double quotes inside double quotes. You can use """ instead:

>     rdfs:comment """For questions that are part of learning resources (e.g. Quiz), eduQuestionType indicates the format of question being given. Example: "Multiple choice", "Open ended", "Flashcard".""" .

(I checked this at https://www.w3.org/2015/03/ShExValidata/ but there are other similar tools elsewhere)

Reply:
I've fixed this one directly in the proposal branch